### ISSUE TITLE
Suggestion: strip Google Analytics from URL

### ISSUE THREAD

**Issue by tanderson1992** [WazeDev/WME-Place-Harmonizer#440]
Apologies if this has already been selected. I didn't see it in the open or closed issues.

When correcting a pasted URL, especially from gmaps, remove trailing analytics information automatically.

Example: `https://www.shottenkirkchrysler.com/?utm_source=google&utm_medium=organic&utm_campaign=googlemybusiness` becomes `https://www.shottenkirkchrysler.com`.

**Comment by jm6087** [comment]
I don't think this is necessary since the URL should come from the actual company website and not from the Google search or Gmap results.

**Comment by tanderson1992** [comment]
With Waze and Google staff both saying it's perfectly fine to copy directly from gmaps at the SER meetup in March, I've been doing a lot of copying from maps. Goes against what we were told even after Google bought Waze way back when, but apparently it's ok now.

**Comment by jm6087** [comment]
Ultimately it is MoM's decision on what he does with his script, but since this is not national guidance I would no recommend changing the script.   If SER has changed their guidance and MoM has the ability to narrow down the use to SER only, then that would be fine IMO.
Totally agree, MoM does a fantaboulas job with this script.

**Comment by mapomatic** [comment]
I'm going to go ahead with this in beta. Since some are leaning more toward using the specific place URL vs the top-level URL and the GMaps link often takes you directly to the place URL (with all the analytics stuff added), it makes some sense. We can test in beta and decide if it's worth it.  Note: this isn't a recommendation to use the place URL.  It'll only show a warning if analytics are in the URL and prompt to strip them.
